### ISSUE TITLE
PowerShell is not opening windows toolbox

### ISSUE THREAD

**Issue by hhwwaaffii** [windowtoolbox/powershell-windows-toolbox#19]
Following procedure did work a couple of times but it doesn't work anymore as I try to open it now. The PowerShell is not opening the toolbox and closes the window on its own after a while.

**Comment by Janmm14** [comment]
Malware - Especially english windows installs at risk, see #32

**Comment by tecxpart** [comment]
#32 is not there. I can see up to #27

**Comment by Janmm14** [comment]
@tecxpart Yes, the issues were deleted, look now here: https://github.com/pabumake/windowToolboxMalware-Removal
